dataprovider/src/main/java/cm/aptoide/pt/dataprovider/ws/v7/home/GetHomeBundlesRequest.java

```diff
@@ -0,0 +1,163 @@
+package cm.aptoide.pt.dataprovider.ws.v7.home;
+
+import android.content.SharedPreferences;
+import android.content.res.Resources;
+import android.net.ConnectivityManager;
+import android.view.WindowManager;
+import cm.aptoide.pt.dataprovider.interfaces.TokenInvalidator;
+import cm.aptoide.pt.dataprovider.model.v7.GetStoreWidgets;
+import cm.aptoide.pt.dataprovider.model.v7.Type;
+import cm.aptoide.pt.dataprovider.ws.BodyInterceptor;
+import cm.aptoide.pt.dataprovider.ws.v2.aptwords.AdsApplicationVersionCodeProvider;
+import cm.aptoide.pt.dataprovider.ws.v7.BaseBody;
+import cm.aptoide.pt.dataprovider.ws.v7.BaseRequestWithStore;
+import cm.aptoide.pt.dataprovider.ws.v7.Endless;
+import cm.aptoide.pt.dataprovider.ws.v7.V7;
+import cm.aptoide.pt.dataprovider.ws.v7.WSWidgetsUtils;
+import cm.aptoide.pt.dataprovider.ws.v7.store.StoreContext;
+import cm.aptoide.pt.dataprovider.ws.v7.store.WidgetsArgs;
+import java.util.List;
+import okhttp3.OkHttpClient;
+import retrofit2.Converter;
+import rx.Observable;
+import rx.schedulers.Schedulers;
+
+/**
+ * Created by jdandrade on 08/03/2018.
+ */
+
+public class GetHomeBundlesRequest extends V7<GetStoreWidgets, GetHomeBundlesRequest.Body> {
```

Comment:
Could be private?

Reply:
This can't be private and usually requests should always have maximum scope, but the constructor that is used inside can be private since it is only used inside and I think you are talking about that one